### PR TITLE
Fix for add description to the adapters 

### PR DESCRIPTION
**Pull request by thiliA**

### CODE REVIEW

**Comment by thiliA** [comment]
The above PR also contains a fix for file-tail receiver key null check. (commit d5c2953)

File: components/event-receiver/org.wso2.carbon.event.receiver.admin/src/main/java/org/wso2/carbon/event/receiver/admin/InputAdapterConfigurationDto.java [hunk 1]
```diff
@@ -48,4 +50,14 @@ public void setSupportedMessageFormats(String[] supportedMessageFormats) {
...
     }
 
+    public String getUsageTips() {
+        return usageTips;
+    }
+
+    public void setUsageTips(String usageTips) {
+        this.usageTips = usageTips;
+    }
+
```
Comment (ramindu90) [inline components/event-receiver/org.wso2.carbon.event.receiver.admin/src/main/java/org/wso2/carbon/event/receiver/admin/InputAdapterConfigurationDto.java]: Lets remove unnecessary new lines.

File: components/event-receiver/event-input-adapters/org.wso2.carbon.event.input.adapter.websocket.local/src/main/java/org/wso2/carbon/event/input/adapter/websocket/local/WebsocketLocalEventAdapterFactory.java [hunk 2]
```diff
@@ -45,6 +45,11 @@ public String getType() {
     }
 
     @Override
+    public String getUsageTips() {
+        return WebsocketLocalEventAdapterConstants.ADAPTOR_USAGE_TIPS_WEBSOCKET_LOCAL;
```
Comment (lasanthafdo) [inline components/event-receiver/event-input-adapters/org.wso2.carbon.event.input.adapter.websocket.local/src/main/java/org/wso2/carbon/event/input/adapter/websocket/local/WebsocketLocalEventAdapterFactory.java]: @thiliA 
Why have we hard coded the usage tip for websocket local while it is taken from the resource file for other adapters? If there is no technical limitation, let's keep it consistent across all adapters. Keeping the message in the resource file is the correct approach.

File: components/event-receiver/org.wso2.carbon.event.receiver.admin/src/main/java/org/wso2/carbon/event/receiver/admin/InputAdapterConfigurationDto.java (hunk 1 above)
Comment (thiliA) [inline components/event-receiver/org.wso2.carbon.event.receiver.admin/src/main/java/org/wso2/carbon/event/receiver/admin/InputAdapterConfigurationDto.java]: @ramindu90 
Fixed in https://github.com/wso2/carbon-analytics-common/pull/142

File: components/event-receiver/event-input-adapters/org.wso2.carbon.event.input.adapter.websocket.local/src/main/java/org/wso2/carbon/event/input/adapter/websocket/local/WebsocketLocalEventAdapterFactory.java (hunk 2 above)
Comment (thiliA) [inline components/event-receiver/event-input-adapters/org.wso2.carbon.event.input.adapter.websocket.local/src/main/java/org/wso2/carbon/event/input/adapter/websocket/local/WebsocketLocalEventAdapterFactory.java]: @lasanthafdo 
Yes, we can do that and thank you for pointing it out. Fixed in PR: https://github.com/wso2/carbon-analytics-common/pull/144

**Pull request merged**